<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuBarMovePane</class>
 <widget class="QFrame" name="MenuBarMovePane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="Icon">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="PlaceHolder"/>
   </item>
   <item>
    <widget class="QPushButton" name="BtnMinimize">
     <property name="text">
      <string>Minimize</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="BtnMaximize">
     <property name="text">
      <string>Maximize</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="BtnClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
